tests/sites/missing/devices/AHU-22/expected/errors.map

```diff
@@ -1,4 +1,4 @@
 Exceptions for AHU-22
-  While converting device config
```

Comment:
ideally we'd find a way to preserve this context so that it's easier to diagnose. E.g., something like:

```
Exceptions for AHU-22
  While converting device config
    While converting point filter_alarm_pressure_status
      protocol ref FAXLE12.present_value does not match expression bacnet://[1-9][0-9]*/([A-Z]{2,4}):([1-9][0-9]*)(#[_a-z]+)?
      protocol ref MCN8.present_value does not match expression bacnet://[1-9][0-9]*/([A-Z]{2,4}):([1-9][0-9]*)(#[_a-z]+)?
      protocol ref AI2.differential does not match expression bacnet://[1-9][0-9]*/([A-Z]{2,4}):([1-9][0-9]*)(#[_a-z]+)?
 ```
E.g. consider using the (defined in the code somewhere) `ExceptionList` class, and have that returned by `getSchemaViolations()` (instead of returning the direct list of exceptions), with the appropriate context (converting config, point names, etc...).  I _think_ that should result in the right error output, or at least close enough.  There might need to be a tweak to the error reporting function on what happens when it sees an ExceptionList -- but that should be relatively straightforward.